Kill.py

```diff
@@ -0,0 +1,59 @@
+""" Kill Process Extension """
+
+import os
+import getpass
+from albertv0 import *
+from subprocess import Popen, PIPE
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Kill Process"
+__version__ = "1.0"
+__trigger__ = "kill"
+__author__ = "Benedict Dudel"
+__dependencies__ = []
+
+def handleQuery(query):
+    if query.isTriggered:
+        results = []
+        for process in getProcesses(query.string.strip()):
+            results.append(
+                Item(
+                    id=process['id'],
+                    icon=iconLookup(process['name']),
```

Comment:
Maybe [*process-stop*](https://standards.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html#names) makes sense. What do you think? Looks somewhat like this on my system 

![image](https://user-images.githubusercontent.com/3252315/32414286-c72cc572-c223-11e7-900e-0c9c4a9af947.png)